ci/kokoro/ubuntu/build.sh

```diff
@@ -16,36 +16,100 @@
 
 set -eu
 
-export GOOGLE_APPLICATION_CREDENTIALS="${KOKORO_GFILE_DIR}/service-account.json"
+echo "Reading CI secret configuration parameters."
 source "${KOKORO_GFILE_DIR}/test-configuration.sh"
 
-echo "Getting cbt tool"
-export GOPATH="${KOKORO_ROOT}/golang"
-go get -u cloud.google.com/go/bigtable/cmd/cbt
-
-echo "Getting python dependencies"
-sudo python2 -m pip install gunicorn httpbin
-
 echo "Running build and tests"
 cd "$(dirname $0)/../../.."
 readonly PROJECT_ROOT="${PWD}"
 
-git submodule update --init --recursive
-cmake -DCMAKE_BUILD_TYPE=Release -H. -Bbuild-output
-cmake --build build-output -- -j $(nproc)
 
-cd build-output
-ctest --output-on-failure
+cat >>kokoro-bazelrc <<_EOF_
+# Set flags for uploading to BES without Remote Build Execution.
+startup --host_jvm_args=-Dbazel.DigestFunction=SHA256
+build:results-local --remote_cache=remotebuildexecution.googleapis.com
+build:results-local --spawn_strategy=local
+build:results-local --remote_timeout=3600
+build:results-local --bes_backend="buildeventservice.googleapis.com"
+build:results-local --bes_best_effort=false
+build:results-local --bes_timeout=10m
+build:results-local --tls_enabled=true
+build:results-local --auth_enabled=true
+build:results-local --auth_scope=https://www.googleapis.com/auth/cloud-source-tools
+build:results-local --experimental_remote_spawn_cache
+_EOF_
+
+# First build and run the unit tests.
+INVOCATION_ID="$(python -c 'import uuid; print uuid.uuid4()')"
+echo "Configure and start Bazel: " ${INVOCATION_ID}
+echo "================================================================"
+echo "    https://source.cloud.google.com/results/invocations/${INVOCATION_ID}"
```

Comment:
Any reason for the URL indentation, or can we make it flush-left?

Reply:
Fixed (also it was easier to find in the log).